apps/services/auth/api/src/app/modules/delegations/delegations.controller.ts

```diff
@@ -14,10 +14,12 @@ import {
   ScopesGuard,
 } from '@island.is/auth-nest-tools'
 import type { User } from '@island.is/auth-nest-tools'
+import { Audit } from '@island.is/nest/audit'
 
 @UseGuards(IdsUserGuard, ScopesGuard)
 @ApiTags('delegations')
 @Controller('delegations')
+@Audit({ namespace: '@island.is/api/delegations' })
```

Comment:
Hmm okey maybe I am misunderstanding.
However I put this in delegations.controller since one of its methods is using
```
return this.delegationsService.findAllIncoming(user)
```
which uses it like so
```
      this.auditService.auditSystem({
        action: 'deceasedDelegation',
        resources: deceasedDelegations.map(({ id }) => id).filter(isDefined),
      })
```
The logic I wrote in the `auditSystem` will error if namespace is missing, i.e. in this.checkNamespace()
```

```
  private formatSystemMessage({
    namespace = this.defaultNamespace,
    action,
    resources,
    meta,
  }: AuditSystemMessage) {
    this.checkNameSpace(namespace)
    const namespacePostfix = `${namespace}/system`
    const message = {
      namespace: namespacePostfix,
      action: `${namespacePostfix}#${action}`,
      resources: isString(resources) ? [resources] : resources,
      meta,
      appVersion: process.env.APP_VERSION,
    }

    return this.useDevLogger ? { message: 'Audit record', ...message } : message
  }
```

Would you rather want me to pass in a namespace here?

```
    this.auditService.auditSystem({
        action: 'deceasedDelegation',
        resources: deceasedDelegations.map(({ id }) => id).filter(isDefined),
        namespace: 'Some namespace'
      })
```